examples/sinatra/app.rb

```diff
@@ -21,6 +21,25 @@
   }
 end
 
+get '/dynamic-share' do
+  scenario = Yoti::DynamicSharingService::DynamicScenario.builder.with_policy(
+    Yoti::DynamicSharingService::DynamicPolicy.builder
+    .with_full_name
+    .with_age_over(18)
+    .with_pin_auth
+    .build
+  )
+                                                         .with_callback_endpoint('/profile')
```

Comment:
Fixing the indentation causes a Travis failure: https://travis-ci.com/getyoti/yoti-ruby-sdk/jobs/246199384

Going to refactor these lines to avoid the problem.